Library/Formula/crystax-ndk.rb

```diff
@@ -17,20 +17,15 @@ class CrystaxNdk < Formula
   depends_on "android-sdk" => :recommended
 
   conflicts_with "android-ndk",
-    :because => "both install `ndk-build`, `ndk-gdb` and `ndk-stack` binaries"
+    :because => "both install `ndk-build`, `ndk-gdb`, `ndk-stack`, `ndk-depends` and `ndk-which` binaries"
 
   def install
     bin.mkpath
 
-    if MacOS.prefer_64_bit?
-      arch = :x86_64
-    else
-      arch = :x86
-    end
+    # Cleanup some files since "brew audit" complains they're linked against system OpenSSL
+    %w{_hashlib.so _ssl.so}.each { |file| rm_f Dir.glob("prebuilt/darwin-*/lib/python*/lib-dynload/#{file}") }
```

Comment:
Right, this will not be fixed by adding `depends_on "openssl"`, since it's already prebuilt binaries.

However, it's not bad removing those files - in fact, they aren't used, and were added into CrystaX NDK distribution by mistake. There is no need in SSL in prebuilt python. In next patch (10.3.1?) they will be removed from CrystaX NDK anyway.